aws_topology/stackstate_checks/aws_topology/aws_topology.py

```diff
@@ -161,17 +181,18 @@ def get_topology(self, instance_info, aws_client):
                     }
                     self.event(event)
                     errors.append("API %s ended with exception: %s %s" % (spec["api"], str(e), traceback.format_exc()))
-            # TODO this should be for tests, in production these relations should not be sent out
-            agent_proxy.finalize_account_topology()
-            self.components_seen = agent_proxy.components_seen
-            if len(errors) > 0:
-                raise Exception("get_topology gave following exceptions: %s" % ", ".join(errors))
+        # TODO this should be for tests, in production these relations should not be sent out
+        self.log.info('Finalize FULL scan (#components = {})'.format(len(agent_proxy.components_seen)))
```

Comment:
Will this print all of the components that was seen? In that case I would make it debug log

Reply:
Only the number of components of this scan will be shown